packages/ui5-application-writer/src/types.ts

```diff
@@ -12,39 +12,38 @@ export interface Package {
 
 export interface App {
     id: string;
-    uri?: string; // What is this?
-    version?: string; // What is this?
+    version?: string;
     title?: string;
     description?: string;
     baseComponent?: string; // UI5 Component `required` by Component.js
-    flpAppId?: string;
     startFile?: string;
     localStartFile?: string;
 }
 
+export type UI5Framework = 'SAPUI5' | 'OpenUI5';
+
 export interface UI5 {
-    framework?: 'SAPUI5' | 'OpenUI5';
-    frameworkUrl?: string;
-    minUI5Version?: string;
-    version?: string;
-    localVersion?: string;
-    typesVersion?: string;
-    descriptorVersion?: string;
-    ui5Libs?: string | string[];
-    ui5Theme?: string;
-    directLoad?: boolean;
```

Comment:
Why remove `directLoad`?

Reply:
It moved to `ProxyUIConfig` in `ui5-config/types`